1.26/Dockerfile.c9s

```diff
@@ -1,89 +0,0 @@
-FROM quay.io/sclorg/s2i-core-c9s:c9s
```

Comment:
Keep the Dockerfiles for RHEL/Centos Stream versions we still maintain. Otherwise we just remove all the history that we tried saving by doing the git mv dance.
Removing c8s is fine as we do not expect to build any images for c8s anymore.

Reply:
Fixed by https://github.com/sclorg/nginx-container/pull/297/commits/743fe952ed7a2e9e8c5796108340ed685d91b64e